src/arch/x86/lib/cpu_performance/cpu_info.h

```diff
@@ -1,16 +1,9 @@
 #ifndef X86_CPU_INFO_H_
 #define X86_CPU_INFO_H_
 
-struct cpu_feature {
-	char name[13];
-	unsigned int val; 
-};
+#define FEATURE_NAME_LEN 32
```

Comment:
`FEATURE_NAME_LEN` can either be defined here or in `include/hal/cpu_info.h`.

If we let it be here, then the name length can be different for every Arch. What do you think will be better?

Reply:
I think it's better to define it big enough for all architectures.

Reply:
Alright, if there is anything else that needs to change with this PR, let me know. I will create another fixup commit accordingly. Thanks!

Reply:
Fixed.